ckanext/canada/tables/choices/qpnotes_minister.yaml

```diff
@@ -0,0 +1,35 @@
+M101: Justin Trudeau
+M102: Ralph Goodale
+M103: Lawrence MacAulay
+M104: Carolyn Bennett
+M105: Dominic LeBlanc
+M106: Navdeep Bains
+M107: Bill Morneau
+M108: Chrystia Freeland
+M109: Jean-Yves Duclos
+M110: Marc Garneau
+M111: Marie-Claude Bibeau
+M112: Jim Carr
+M113: Mélanie Joly
+M114: Diane Lebouthillier
+M115: Catherine McKenna
+M116: Harjit S. Sajjan
+M117: Amarjeet Sohi
+M118: Maryam Monsef
+M119: Carla Qualtrough
+M120: Kirsty Duncan
+M121: Patty Hajdu
+M122: Bardish Chagger
+M123: François-Philippe Champagne
+M124: Karina Gould
+M125: Ahmed Hussen
+M126: Ginette Petitpas Taylor
+M127: Seamus O' Regan
+M128: Pablo Rodriguez
+M129: Bill Blair
+M130: Mary Ng
+M131: Filomena Tassi
+M132: Jonathan Wilkinson
+M133: David Lametti
+M134: Bernadette Jordan
+M135: Joyce Murray
```

Comment:
Are these ids from the source CSV? I think we need to store the actual names in our table instead of an opaque minister ID (the keys are what's stored not the values)